front-end/app/checkout/checkout.component.html

```diff
@@ -29,7 +29,7 @@
             </div>
 
             <h3>Order notes</h3>
-            <textarea class="form-control" rows="3" name="order_notes" [(ngModel)]="basket.orderNotes"></textarea>
+            <textarea class="form-control" rows="3" name="order_notes" [(ngModel)]="basket.orderNotes" placeholder="i.e. any adjustments to items or a arrange delivery for later..."></textarea>
```

Comment:
The issue didn't have the correct text... it wasn't even correct English.

A more correct message would say "Enter any special instructions for your order. For example swapping/removing a pizzas toppings or instructing that you would like to place an order to be delivered at a specific time."